dev/source/docs/building-setup-windows10.rst

```diff
@@ -18,7 +18,10 @@ Setup steps
     Starting with the Fall Creators Update (Windows 10 version 1803), enabling the Developer Mode is not longer required for using WSL.
 
 #. Enable Ubuntu on Windows which includes the following steps (original `How-To Geek's instructions here <http://www.howtogeek.com/249966/how-to-install-and-use-the-linux-bash-shell-on-windows-10/>`__):
-In case of struggle, please refer to Microsoft official documentation : `<https://docs.microsoft.com/en-us/windows/wsl/install-win10>`__
+
+    .. note::
+
+        In case of struggle, please refer to Microsoft official documentation : `<https://docs.microsoft.com/en-us/windows/wsl/install-win10>`__
```

Comment:
An existing error but more a more natural sentence would be, "In case of troubles, please refer to the official documentation from Microsoft : xxx"